clinica/pipelines/statistics_volume/statistics_volume_pipeline.py

```diff
@@ -0,0 +1,347 @@
+
+__author__ = "Arnaud Marcoux"
+__copyright__ = "Copyright 2016-2019 The Aramis Lab Team"
+__credits__ = ["Arnaud Marcoux"]
+__license__ = "See LICENSE.txt file"
+__version__ = "0.3.0"
+__maintainer__ = "Arnaud Marcoux"
+__email__ = "arnaud.marcoux@icm-institute.org"
+__status__ = "Development"
+
+
+import clinica.pipelines.engine as cpe
+
+
+class StatisticsVolume(cpe.Pipeline):
+    """Statistics_Volume SHORT DESCRIPTION.
+
+
+    Args:
+        input_dir: A BIDS directory.
+        output_dir: An empty output directory where CAPS structured data will be written.
+        subjects_sessions_list: The Subjects-Sessions list file (in .tsv format).
+
+    Returns:
+        A clinica pipeline object containing the Statistics_Volume pipeline.
+
+    Raises:
+
+
+    """
+
+    def check_custom_dependencies(self):
+        """Check dependencies that can not be listed in the `info.json` file.
+        """
+        pass
+
+    def get_input_fields(self):
+        """Specify the list of possible inputs of this pipeline.
+
+        Returns:
+            A list of (string) input fields name.
+        """
+
+        return ['input_files']
+
+    def get_output_fields(self):
+        """Specify the list of possible outputs of this pipeline.
+
+        Returns:
+            A list of (string) output fields name.
+        """
+
+        return ['spmT_0001',
+                'spmT_0002',
```

Comment:
Can you confirm that `spmT_0001` is for `<group_1> < <group_2>` and `spmT_0001` is for `<group_1> < <group_2>` (or vice-versa)

Reply:
`spmT_0001` is associated with the task of the first contrast set so yes.